include/irr/asset/IRenderpassIndependentPipeline.h

```diff
@@ -27,14 +28,33 @@ enum E_PRIMITIVE_TOPOLOGY
     EPT_PATCH_LIST = 10
 };
 
-struct SVertexInputParams
+enum E_VERTEX_INPUT_RATE
+{
+    EVIR_PER_VERTEX = 0,
+    EVIR_PER_INSTANCE = 1
+};
+
+struct SVertexInputAttribParams
 {
-    bool enabled;
-    bool perInstance;
     E_FORMAT format;
```

Comment:
missing location and binding field (only one of tem can be implicit)

Reply:
you dont have a `binding` field, that's necessary

Reply:
binding field is redundant because index in the array is supposed to denote binding num

Reply:
nope, the index in the array denotes the `location` not the binding (which buffer binding provides the attribute for the current location)

Reply:
resolved